unleashTests/Services/ToggleServiceSpec.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ToggleServiceSpec.swift
+//  UnleashTests
+//
+//  Copyright © 2019 Silvercar. All rights reserved.
+//
+
+import Foundation
+import Nimble
+import OHHTTPStubs.Swift
+import PromiseKit
+import Quick
+
+@testable import Unleash
+
+class ToggleServiceSpec : QuickSpec {
+    override func spec() {
+        let appName: String = "test app"
+        let instanceId: String = "123"
+        var service: ToggleService {
+            return ToggleService(appName: appName, instanceId: instanceId)
```

Comment:
I thought one liners didn't need an explicit return. Maybe something I'm overly sensitive to because of Ruby.

Reply:
I'm not sure properties support that like closures do.